src/pages/ReimbursementAccount/ReimbursementAccountPage.js

```diff
@@ -125,7 +124,7 @@ class ReimbursementAccountPage extends React.Component {
      * @returns {*}
      */
     getDefaultStateForField(fieldName, defaultValue = '') {
-        return ReimbursementAccountUtils.getDefaultStateForField(this.props.reimbursementAccountDraft, this.props.reimbursementAccount, fieldName, defaultValue);
+        return lodashGet(this.props.reimbursementAccount, ['achData', fieldName], defaultValue);
```

Comment:
This change led to regression. We should prefer to draft that got deleted, more details [here](https://github.com/Expensify/App/issues/16938#issuecomment-1513403194)